src/debug.c

```diff
@@ -778,6 +805,16 @@ NULL
     {
         server.active_expire_enabled = atoi(c->argv[2]->ptr);
         addReply(c,shared.ok);
+    } else if (!strcasecmp(c->argv[1]->ptr,"quicklist-packed-threshold") &&
+    c->argc == 3)
+    {
```

Comment:
styling: either we use one line, or if we break the `if` into two lines, we need to indent the conditions and braces differently
```suggestion
    } else if (!strcasecmp(c->argv[1]->ptr,"quicklist-packed-threshold") && c->argc == 3) {
```